search.py

```diff
@@ -27,24 +22,46 @@ def main(user_query, browser):
     soup_link = BeautifulSoup(requests_result.content, "html.parser")
```

Comment:
the task was to get results from the browser, not from requests.get() function
nice try :)

Reply:
additionally, you should put query in query field on google.com website not calling google request.